cmd/oras/internal/display/status/print.go

```diff
@@ -41,11 +42,14 @@ func NewPrinter(out io.Writer) *Printer {
 }
 
 // Println prints objects concurrent-safely with newline.
-func (p *Printer) Println(a ...any) error {
+func (p *Printer) Println(a ...any) {
 	p.lock.Lock()
 	defer p.lock.Unlock()
 	_, err := fmt.Fprintln(p.out, a...)
-	return err
+	if err != nil {
+		err = fmt.Errorf("display output error: %w", err)
+		_, _ = fmt.Fprint(os.Stderr, err)
+	}
 }
```

Comment:
Need to return nil here and add comment to explain why and how the print errors are ignored.

Reply:
I think I've handled all your concerns